### PR TITLE
DB-3986: Render a gradient instead of the Pantheon logo when there is no featured image

### PR DESCRIPTION
**Pull request by enZane**
## What changes were made?
- In all the starters, removed the pantheon logo as the default featured image
- Used a generic gradient instead of that pantheon logo in the cards

## Where were the changes made?

- Next-drupal-starter
- Next-WordPress-starter
- Gatsby-WordPress-starter

## How have the changes been tested?
- Tested in every starter

## Additional information
- We should consider a react kit for these components? because is used in Gatsby and Next

Don't forget to
[add a changeset](https://github.com/pantheon-systems/decoupled-kit-js#generating-a-changeset)
if needed!

### CODE REVIEW

File: .changeset/fluffy-peaches-confess.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+'@pantheon-systems/gatsby-wordpress-starter': minor
+'@pantheon-systems/next-drupal-starter': minor
+'@pantheon-systems/next-wordpress-starter': minor
+---
+
+Remove pantheon logo for cards that doesn't have featured image and use a
+generic gradient instead
```
Comment (CobyPear) [inline .changeset/fluffy-peaches-confess.md:8]:
```suggestion
Remove pantheon logo for cards that don't have featured image, and use a
generic gradient instead
```

File: starters/gatsby-wordpress-starter/src/components/grid.jsx [hunk 2]
```diff
@@ -1,8 +1,12 @@
 import React from 'react'
 
-import { GatsbyImage, StaticImage } from 'gatsby-plugin-image'
+import { GatsbyImage } from 'gatsby-plugin-image'
 import { Link } from 'gatsby'
 
+const GradientPlaceholder = () => (
+	<div className="w-full h-full bg-gradient-to-b from-blue-400 to-blue-500" />
```
Comment (CobyPear) [inline starters/gatsby-wordpress-starter/src/components/grid.jsx:7]: 🤔 I am wondering if the gradient is too subtle? maybe `from-blue-100 to-blue-500`?

@backlineint, what do you think?
Current:
image


Suggested:
image


Also, not sold on the blue, but it's fine :)

**Comment by CobyPear** [comment]
> We should consider a react kit for these components? because is used in Gatsby and Next

Agreed, this is a good candidate for `react-kit`, but I wonder if it's too much effort right now and it should be extracted out later when we have more to share between the react based stuff.

File: starters/gatsby-wordpress-starter/src/components/grid.jsx (hunk 2 above)
Comment (backlineint) [reply starters/gatsby-wordpress-starter/src/components/grid.jsx]: @CobyPear @enZane Agree that the gradient change makes this pop a little bit more. The blue is a little boring, but the previous gradient was a little ugly, so I'll take this :)

**Comment by backlineint** [comment]
> > We should consider a react kit for these components? because is used in Gatsby and Next
> 
> Agreed, this is a good candidate for `react-kit`, but I wonder if it's too much effort right now and it should be extracted out later when we have more to share between the react based stuff.

Also agree this should be follow up work.

**Comment by backlineint** [comment]
@enZane and @CobyPear I did notice that on this branch, there seems to be a regression in the 'welcome to Next.js' heading.
Screenshot at Sep 26 12-48-41
Double checked and don't see it on canary.

**Comment by CobyPear** [comment]
> @enZane and @CobyPear I did notice that on this branch, there seems to be a regression in the 'welcome to Next.js' heading. Screenshot at Sep 26 12-48-41 Double checked and don't see it on canary.

@backlineint This is part of the regressions fixed by adding the prefix to the tailwind classes in nextjs-kit`

**Review by CobyPear — APPROVED** [review]

**Pull request merged**